geopandas/io/tests/test_sql.py

```diff
@@ -365,16 +364,12 @@ def test_write_postgis_append_when_table_exists(self, df_nybb):
 
         except AssertionError as e:
```

Comment:
you can remove the try / except completely now, I think, since this except is just reraising the assertion error now

Reply:
(and similarly for some of the other tests)

Reply:
Omg, unbelievable that I missed these 🙈 Well, good to have more eyes on the codes. Fixed these now for all tests.